crengine/include/cssdef.h

```diff
@@ -414,4 +414,27 @@ typedef struct css_length_tag {
     lUInt32 pack() { return (lUInt32)type + (((lUInt32)value)<<4); }
 } css_length_t;
 
+// Support
+
+// Integer division with rounding to nearest.
+//
+// This does not bother to 'round to nearest integer' in the case of a tie.
+static inline int roundi(int n, int divisor)
```

Comment:
@poire-z is not wrong, divisor ought to be an `unsigned int` to avoid potential UB.

Reply:
Also, the C99 convention for those is to lead with the return type, so I'd rename this `iroundi` ;).